static/elements/chromedash-feature.js

```diff
@@ -0,0 +1,492 @@
+import {LitElement, html} from 'https://unpkg.com/@polymer/lit-element@latest/lit-element.js?module';
+import 'https://unpkg.com/@polymer/iron-icon/iron-icon.js?module';
+import '/static/elements/chromedash-color-status.js';
+
+const MAX_STANDARDS_VAL = 6;
+const MAX_VENDOR_VIEW = 7;
+const MAX_WEBDEV_VIEW = 6;
+const MAX_RISK = MAX_VENDOR_VIEW + MAX_WEBDEV_VIEW + MAX_STANDARDS_VAL;
+
+const IS_PUSH_NOTIFIER_ENABLED = window.PushNotifier.GRANTED_ACCESS;
+const IS_PUSH_NOTIFIER_SUPPORTED =
+    window.PushNotifier && window.PushNotifier.SUPPORTS_NOTIFICATIONS;
+
+class ChromedashFeature extends LitElement {
+  static get properties() {
+    return {
+      feature: {type: Object}, // From attribute
```

Comment:
Any change you'd use TypeScript in the future so we can see what type of object this is?

Reply:
Good suggestion!